pkg/eval/mods/file/file.go

```diff
@@ -124,3 +127,55 @@ func prclose(p vals.Pipe) error {
 func pwclose(p vals.Pipe) error {
 	return p.WriteEnd.Close()
 }
+
+//elvdoc:fn truncate
+//
+// ```elvish
+// file:truncate $filename $size
+// ```
+// Truncate changes the size of
+// the named file. If the file is
+// a symbolic link, it changes the
+// size of the link's target.
+//
+// If there is an error it will
+// be of type *PathError.
```

Comment:
This seems to be copied verbatim from https://pkg.go.dev/os#Truncate; Go docs have different conventions to elvdocs and I'll amend this.